rust_src/src/javascript.rs

```diff
@@ -442,6 +442,15 @@ macro_rules! unproxy {
     }};
 }
 
+const REVERSE_PROXY_MARKER: i64 = 0x22fafd;
+
+macro_rules! make_reverse_proxy {
+    ($scope:expr, $lisp:expr) => {{
+        let marker = unsafe { emacs::bindings::make_int(REVERSE_PROXY_MARKER) };
```

Comment:
LispObject::from_fixnum(REVERSE_PROXY_MARKER) ?

Reply:
Oh I just realized there is a difference
```c
INLINE Lisp_Object
make_int (intmax_t n)
{
  return FIXNUM_OVERFLOW_P (n) ? make_bigint (n) : make_fixnum (n);
}
```
Do you think it makes sense to add a method for this ?

Reply:
I think this is worth addressing, but not needed for this PR. This implementation wouldn't support BIGNUMs on the JS side, so it's fine to just use the fixnum API since it will be simpler code.